docs/paas-plus/dev/integrate-spryker-code-upgrader.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Integrate Spryker Code Upgrader into your development process
+description: How to integrate Spryker Code Upgrader into your development process
+template: concept-topic-template
+---
+
+When integrating Spryker Code Upgrader into your development process it is important to consider your development process and integrate it correctly, here we would like to briefly explain how Spryker Code Upgrader can be integrated into your Scrum development process and Git flow.
+
+![Spryker CI projects](https://spryker.s3.eu-central-1.amazonaws.com/docs/paas%2B/dev/integrate-spryker-code-upgrader.md/spryker-code-upgrader-integration-diagramm.png)
+
+## Scrum
+
+* Sprint Planning: The team can review the upgrade suggestions as pull requests in your repository hosting service and prioritize the software upgrades that need to be installed in the upcoming sprint.
+
+* Sprint Execution: During the sprint, the team can review, validate, modify and merge the upgrades using the new process offered by Spryker Code Upgrader.
+
+This approach will allow you to break up bigger list of upgrades into a list of smaller pieces, each smaller piece can be processed within a sprint ensuring continuity of this process.
+
+## Git Flow
+
+* When connecting Spryker Code Upgrader to your code hosting service you need to pick a base branch, that Spryker Code Upgrader will use. It can be Develop Branch.
+
+* When Spryker Code Upgrader runs, it will pull the configured branch, analyze you installation, upgrade code and create a pull request.
+
+* This pull request should be processed by the team as covered above, at the end it is either merged or closed. If Spryker Code Upgrader detects an open pull request, it will not suggest a new one.
```

Comment:
How should I break down updates into smaller chunks if the upgrader creates only one PR with all the updates?